teach/templates/clubs/email/create_club_staff/body.txt

```diff
@@ -0,0 +1,14 @@
+{{ username }} would like to add their Club to the map.
```

Comment:
is `username` the user's full name they filled out in the application form, or the webmaker user?